deploy/deploy-centralvenv.sh

```diff
@@ -0,0 +1,1000 @@
+                                     The full list of installable services: "reqmgr2 reqmgr2ms workqueue reqmon t0_reqmon"
+                                     (in double quotes and space separated e.g. "rqmgr2 reqmgr2ms")
+                                     (pip based package version syntax is also acceptable e.g. "wmcore==2.0.0")
+      -i <pypi-index>                The pypi index to use (i.e. prod or test) [Default: prod - pointing to https://pypi.org/simple/]
+      -h <help>                      Provides help to the current script
+
+    # Example: Deploy WMCore central services version 2.0.3rc1 linked with 'cmsweb-test1.cern.ch' as a frontend from 'test' pypi index
+    #          at destination /data/tmp/WMCore.venv3/ and using 'Some security string' as a security string for operationss at runtime:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -i test -l wmcore==2.0.3rc1 -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Same as above, but do not cleanup deployment area and reuse it from previous installtion - usefull for testing behaviour
+    #          of different versions during development or mix running from source and from pypi installed packages.
+    #          NOTE: The 'current' link must point to the proper deployment area e.g. either to 'srv/master' for running from source
+    #                or to 'srv/2.0.3rc1' for running from pypi installed package):
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -n -i test -l wmcore==2.0.3rc1 -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Deploy WMCore central services from source repository, use tag 2.0.0.pre3, linked with 'cmsweb-test1.cern.ch' as a frontend
+    #          at destination /data/tmp/WMCore.venv3/ and using 'Some security string' as a security string for operationss at runtime:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -s -t 2.0.0.pre3 -p "10003 9998" -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Same as above, but assume 'Yes' to all questions. To be used in order to chose the default flow and rely only
+    #          on the pameters set for configuring the deployment steps. This will avoid human intervention during deployment:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -y -s -t 2.0.0.pre3 -p "10003 9998" -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+
+    # DEPENDENCIES: All WMCore packages have OS or external libraries/packages dependencies, which are not having a pypi equivalent.
+    #               So far those has been resolved through the set of *.spec files maintained at: https://github.com/cms-sw/cmsdist/tree/comp_gcc630
+    #               Here follows the list of all direct (first level) dependencies per service generated from those spec files:
+
+    #               acdcserver: [python3==3.8.2, rotatelogs==2.2.25, couchdb16==1.6.1]
```

Comment:
I have removed the package versions from the above list and rephrased the message so that people do not get the impression they should generate this list themselves.